contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,101 @@
+
+The FMV is decided through an external, very complicated process known as a 409A, and the initial sale price of the share is usually set when the company "goes public" or "IPOs" by being listed on a public stock exchange like the NYSE. This is known as a “liquidation event” and in the simplest terms, the difference between your exercise price and the price of the share you sell in the public market is cash in your pocket. 
+
+The reality of this is much more complicated, as there can be a number of [tax implications](https://secfi.com/learn/exercise-stock-options-tax-implications) depending on the type of option you were granted, when you decide to exercise them relative to when you sell them, and your overall income. You really should talk to an accountant before making any serious decisions so you don’t end up with an unexpectedly massive tax bill.
+
+Making matters more complicated, the majority of standard terms in stock option plans limit your freedom in when and how you can exercise your options.
+
+In the UK, stock options are offered under the tax-advantged [EMI scheme](https://theaccountancycloud.com/blogs/a-simple-guide-to-enterprise-management-incentive-emi#:~:text=The%20Enterprise%20Management%20Incentive%20(EMI,equity%20participation%20in%20the%20business.), which has a different (and usually much lower) strike price than the FMV.
+
+## How PostHog does things differently
+
+|                           | **PostHog**                 | **Most startups**           |
+|---------------------------|-----------------------------|------------------------------|
+| **Vesting Start:**        | On your first day           | After a "probationary period"|
+| **Exercise Window:**      | 10 _years_ after departure  | 90 _days_ after departure    |
+| **Merger or Acquisition** | Double-Trigger acceleration | No acceleration              |
+| **Good Leavers**.         | Includes performance lay-off| Doesn't include performance  |
+
+### Vesting Start
+
+At PostHog, we recognize that you start contributing to the company on day one, so we feel that you should start earning your options on that day as well. Many others make you wait until after a probationary period, often three months, before your options start to vest.
+
+### Exercise Window
+
+Regardless of whether you resign, or are let go for performance reasons, you will have 10 years from the day you were hired to exercise your options. In most companies, you have only three months to exercise your options before you lose them entirely, which can make tax optimization and planning a stressful and difficult process. We also believe that this window allows you the opportunity to watch the company's growth before deciding to exercise your options. One thing to note is that based on IRS law, tax advantaged ISOs convert into NSOs after 3 months, so there may still be an incentive to exercise sooner.
```

Comment:
This may be more obvious to Us folks, but I have no idea what ISOs and NSOs are. Do they need explaining somewhere?